src/main/java/Duke.java

```diff
@@ -1,10 +1,53 @@
-public class Duke {
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Duke  {
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+                + "|____/ \\__\n" +
+                "    },_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        Greet greet = new Greet();
+        greet.sayHello();
+
+        List<Task> taskList = new ArrayList<>();
+
+        String line;
+        Scanner in = new Scanner(System.in);
+        while(true) {
+            line = in.nextLine();
+            if (line.equals("bye")) {
```

Comment:
I think maybe you can also benefit from including the cases 'Bye' as well as 'bye'